spark/changelog.d/18791.added

```diff
@@ -0,0 +1,2 @@
+Add configuration option to disable `stage_id` tag on `spark.job` metrics
+Add configuration option to disable `spark.stage` metrics
```

Comment:
Should prob combine these in a single entry or separate into 2. Idk how it towncrier will compile this.